plugins/palo_alto_cortex_xdr/unit_test/responses/monitor_alerts_two.json.resp

```diff
@@ -0,0 +1,171 @@
+{
+  "reply": {
+    "total_count": 1000,
+    "result_count": 1,
+    "alerts": [
+      {
+        "agent_os_sub_type": "6.3.4201",
+        "fw_app_category": null,
+        "fw_app_id": null,
+        "fw_app_subcategory": null,
+        "fw_app_technology": null,
+        "causality_actor_process_command_line": null,
+        "causality_actor_process_image_md5": null,
+        "causality_actor_process_image_name": null,
+        "causality_actor_process_image_path": null,
+        "causality_actor_process_image_sha256": null,
+        "causality_actor_process_signature_status": "N/A",
+        "causality_actor_process_signature_vendor": null,
+        "causality_actor_causality_id": null,
+        "identity_sub_type": null,
+        "identity_type": null,
+        "operation_name": null,
+        "project": null,
+        "cloud_provider": null,
+        "referenced_resource": null,
+        "resource_sub_type": null,
+        "resource_type": null,
+        "cluster_name": null,
+        "container_id": null,
+        "contains_featured_host": "NO",
+        "contains_featured_ip": "NO",
+        "contains_featured_user": "NO",
+        "action_country": "UNKNOWN",
+        "fw_interface_to": null,
+        "dns_query_name": null,
+        "agent_device_domain": "attractions.conor.com",
+        "fw_email_recipient": null,
+        "fw_email_sender": null,
+        "fw_email_subject": null,
+        "event_type": null,
+        "is_whitelisted": false,
+        "action_file_macro_sha256": null,
+        "action_file_md5": null,
+        "action_file_name": null,
+        "action_file_path": null,
+        "action_file_sha256": null,
+        "fw_device_name": null,
+        "fw_rule_id": null,
+        "fw_rule": null,
+        "fw_serial_number": null,
+        "agent_fqdn": "FL-ATR-DC3-04.attractions.disney.com",
```

Comment:
have these test responses been sanitized?

Reply:
Yes all the data has been changed